pkg/health/server/server.go

```diff
@@ -212,15 +238,17 @@ func (s *Server) runActiveServices() error {
 	// Run it once at the start so we get some initial status
 	s.FetchStatusResponse()
 
-	nodes, _ := s.getNodes()
-	prober := newProber(s, nodes)
+	nodesAdded, _, _ := s.getNodes(false)
```

Comment:
This ignores `nodesRemoved` because it's the first call to `s.getNodes(false)`, which I believe is OK but I think that is worth adding a comment for here, just in case this gets refactored in future.